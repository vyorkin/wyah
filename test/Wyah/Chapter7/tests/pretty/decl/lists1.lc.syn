Decl "indx" (ELam (Var "n") (ELam (Var "xs") (EApp (EVar (Var "head")) (EApp (EApp (EVar (Var "n")) (EVar (Var "tail"))) (EVar (Var "xs"))))))
