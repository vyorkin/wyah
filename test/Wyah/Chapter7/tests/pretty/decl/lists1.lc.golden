let indx = \n -> \xs -> head ((n tail) xs);